guides/source/configuring.md

```diff
@@ -1015,6 +1017,9 @@ text/javascript image/svg+xml application/postscript application/x-shockwave-fla
 
 `config.load_defaults` sets new defaults up to and including the version passed. Such that passing, say, '6.0' also gets the new defaults from every version before it.
 
+#### For '6.2', defaults from previous versions below and:
+- `config.active_support.key_generator_hash_digest_class`: `OpenSSL::Digest::SHA1`
```

Comment:
If the default didn't change we don't need to document here. Those line can be removed.

Reply:
@rafaelfranca I've updated the change here to move the default and document it here properly. Existing apps stay with SHA1 then.

Reply:
Since I now did change the default, I think this need to be kept then.